ACLPlugin/Source/ACLPlugin/Private/AnimCompress_ACL.cpp

```diff
@@ -184,7 +185,12 @@ void UAnimCompress_ACL::PopulateDDCKey(FArchive& Ar)
 	uint32 ForceRebuildVersion = 1;
 	uint16 AlgorithmVersion = get_algorithm_version(AlgorithmType8::UniformlySampled);
 	uint32 SettingsHash = Settings.get_hash();
-
+	
+	for (const FString& MatchName : UAnimationSettings::Get()->KeyEndEffectorsMatchNameArray)
```

Comment:
That is a great idea! Can you add a separate value for the hash to keep it apart and append it to the Ar archive on line 195?